<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsBiot</class>
 <widget class="QWidget" name="SettingsBiot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Biot Options</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mutation Rate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="mutationComboBox">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Nearly none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Very slight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slight</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Moderate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Very high</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Too high</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Regeneration Cost</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="regenCostcomboBox">
       <item>
        <property name="text">
         <string>Nothing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cheap</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Expensive</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Too expensive</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Regeneration Rate</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="regenRateComboBox">
       <item>
        <property name="text">
         <string>Immediate</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Very quick</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Quickly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Normal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Slowly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Very slow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Too slow</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Reproduction</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="reproductionComboBox">
       <item>
        <property name="text">
         <string>Asexual</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sexual</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Both asexual and sexual</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Allow parent to attack child</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="parentAttackCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Allow siblings to battle</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="siblingAttackCheckBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
